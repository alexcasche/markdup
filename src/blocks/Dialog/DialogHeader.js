import styled from 'styled-components'
import { tint } from 'polished'

const DialogHeader = styled.h1`
  margin: 0;
  text-align: left;
  color: ${props => tint(0.25, props.theme.colors.grey)};
  font-weight: bold;
  font-size: 18px;
  letter-spacing: 1px;
  line-height: 28px;
`

export default DialogHeader
