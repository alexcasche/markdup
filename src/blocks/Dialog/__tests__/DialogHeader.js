import DialogHeader from '../DialogHeader'

describe('DialogHeader Element', () => {
  it('renders correctly', () => {
    const tree = render(<DialogHeader />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
