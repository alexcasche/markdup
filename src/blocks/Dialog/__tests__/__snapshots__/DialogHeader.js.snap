// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DialogHeader Element renders correctly 1`] = `
.c0 {
  margin: 0;
  text-align: left;
  color: #eaeaec;
  font-weight: bold;
  font-size: 18px;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  line-height: 28px;
}

<h1
  className="c0"
/>
`;
